hypothesis/via
# Add a malicious block page inspired by the one in ViaHTML

For: https://github.com/hypothesis/checkmate/issues/10

### Testing notes

 * Run `make dev`
 * Observe that `bad.example.net` is marked as malicious in `via/default-blocklist.txt`
 * Visit: http://localhost:9080/http://bad.example.net
 * See the page in all it's glory
